pkg/generators/markers_test.go

```diff
@@ -123,16 +122,24 @@ func TestParseCommentTags(t *testing.T) {
 		},
 		{
 			t:    types.Float64,
-			name: "invalid: invalid value",
+			name: "invalid: non-JSON value",
+			comments: []string{
+				`+k8s:validation:minimum=asdf`,
+			},
+			expectedError: `failed to parse marker comments: failed to parse value for key minimum as JSON: invalid character 'a' looking for beginning of value`,
+		},
+		{
+			t:    types.Float64,
+			name: "invalid: invalid value type",
 			comments: []string{
-				"+k8s:validation:minimum=asdf",
+				`+k8s:validation:minimum="asdf"`,
```

Comment:
Do we have bare strings like in use today that we'll need to migrate?

Reply:
Nope, the `+k8s:validation` prefix is not used anywhere